src/test/ledger/Invariants_test.cpp

```diff
@@ -44,16 +54,20 @@ class Invariants_test : public beast::unit_test::suite
         Precheck const& precheck,
```

Comment:
Minor note: some sort of comment/docstring on this function would be super helpful

Reply:
Added in 527a964